lib/trino-parquet/src/main/java/io/trino/parquet/ParquetTimestampUtils.java

```diff
@@ -63,9 +61,8 @@ public static DecodedTimestamp decodeInt96Timestamp(Binary timestampBinary)
 
     public static DecodedTimestamp decodeInt96Timestamp(long timeOfDayNanos, int julianDay)
     {
-        verify(timeOfDayNanos >= 0 && timeOfDayNanos < NANOSECONDS_PER_DAY, "Invalid timeOfDayNanos: %s", timeOfDayNanos);
```

Comment:
Has this been considered in https://github.com/trinodb/trino/pull/19169?

Reply:
That PR was removing validation from `io.trino.plugin.base.type.DecodedTimestamp#DecodedTimestamp` constructor too which we want to avoid. I don't think that PR attempted anything another than just removing validation.